src/number_theory/padics/padic_norm.lean

```diff
@@ -83,7 +83,7 @@ The p-adic norm of `p` is `1/p` if `p` is prime.
 
 See also `padic_norm.padic_norm_p` for a version that assumes `1 < p`.
 -/
-@[simp] lemma padic_norm_p_of_prime (p : ℕ) [fact p.prime] : padic_norm p p = 1 / p :=
+@[simp] lemma padic_norm_p_of_prime {p : ℕ} [fact p.prime] : padic_norm p p = 1 / p :=
```

Comment:
`p` is already defined, right? And it is already implicit, so no need to redefine it here. Same below.

Reply:
I missed this, thanks.